session/bootstrap.go

```diff
@@ -222,10 +222,11 @@ const (
       	bind_sql text NOT NULL ,
       	default_db text  NOT NULL,
 		status text NOT NULL,
-		create_time timestamp NOT NULL,
-		update_time timestamp NOT NULL,
+		create_time timestamp(3) NOT NULL,
```

Comment:
please do not modify this. Instead you should alter the column definition in the `upgradeToVer29()` operation.

Reply:
upgrade and this both need modify, because when the tidb is first start in the server, the version is 29.